packages/commonwealth/client/scripts/views/pages/Communities/AllTabContent.tsx

```diff
@@ -54,124 +60,203 @@ const AllTabContent: React.FC<AllTabContentProps> = ({
   const launchpadEnabled = useFlag('launchpad');
   const questsEnabled = useFlag('xp');
   const user = useUserStore();
+  const navigate = useCommonNavigate();
 
   return (
     <>
       {launchpadEnabled && (
         <div className="section-container">
-          <TokensList filters={filters} />
+          <div className="heading-container">
+            <CWText type="h2">Tokens</CWText>
+            <div
+              className="link-right"
+              onClick={() => navigate('/explore?tab=tokens')}
+            >
+              <CWText className="link">See all tokens</CWText>
+              <CWIcon iconName="arrowRightPhosphor" className="blue-icon" />
+            </div>
+          </div>
```

Comment:
this piece of code is repated 6 times in this file, we should make it a SeeAll component in separate file

Reply:
this one I will resolve